packages/shared-types/src/ComponentThemeVariables.ts

```diff
@@ -40,8 +39,8 @@ export interface ComponentTheme {
 }
 
 export type AlertTheme = {
-  background: Colors['backgroundLightest']
-  color: Colors['textDarkest']
+  background: string
+  color: string
```

Comment:
Could we type these more exactly like in the old version?

Reply:
Note: after deliberation, the team decided to go with the "old version", so I did a refactor